src/scaladoc/scala/tools/nsc/doc/html/resource/lib/index.js

```diff
@@ -37,7 +37,7 @@ var QueryString = function(key) {
 
 $(document).ready(function() {
     // Clicking #doc-title returns the user to the root package
-    $("#doc-title").click(function() { document.location = toRoot + "index.html" });
+    $("#doc-title").on("click", function() { document.location = toRoot + "index.html" });
```

Comment:
`click` is deprecated?

Reply:
Not deprecated actually.
https://api.jquery.com/click/

This change is done since jQuery migration plugin recommends to use `on` and `trigger`.
`click` or other event handlers are just shorthand for on `on` and `trigger`.
Those mix 2 different concepts.
I think `on`/`trigger` is more clear to convey intent.